ceph/ceph_admin/iscsi.py

```diff
@@ -30,8 +32,11 @@ def apply(self, config: Dict) -> None:
                     - india             # name of the pool
                     - api_user          # name of the API user
                     - api_pass          # password of the api_user.
-                    - trusted_ip_list   # space separate list of IPs
+
                 args:
+                    trusted_ip_list:       #its no more a positional arg in 5.x release
```

Comment:
`#its no more a positional arg in 5.x release`
I think it should be changed now.

Reply:
updated